app/templates/views/organisations/organisation/index.html

```diff
@@ -112,4 +112,10 @@ <h3 class="govuk-!-font-weight-bold govuk-!-font-size-24 govuk-!-margin-bottom-3
     <div class="keyline-block govuk-!-margin-top-2"></div>
   {% endif %}
 
+  <div class="js-stick-at-bottom-when-scrolling">
+    <p class="bottom-gutter">
+      <a href="{{ download_link }}" download="download" class="govuk-link govuk-link--no-visited-state govuk-!-font-weight-bold">Download this report</a>
```

Comment:
In other places where we have download links in a sticky footer (for example after sending a letter) the link isn’t bold:
```suggestion
      <a href="{{ download_link }}" download="download" class="govuk-link govuk-link--no-visited-state">Download this report</a>
```